config/samples/kepler.system_v1alpha1_powermonitor.yaml

```diff
@@ -10,3 +10,8 @@ spec:
   kepler:
     config:
       logLevel: info
+    deployment:
+      security:
+        mode: none
+        allowedSANames:
```

Comment:
I don't think you need to include this because if this is not included, the default will be added depending on cluster type already.